tree/ntuple/v7/test/ntuple_packing.cxx

```diff
@@ -5,20 +5,23 @@
 #include <cstring> // for memcmp
 #include <limits>
 #include <type_traits>
+#include "ROOT/RColumnElement.hxx"
 #include <utility>
 
-template <typename PodT, typename NarrowT, ROOT::Experimental::EColumnType ColumnT>
+using ROOT::Experimental::Internal::RColumnElement;
```

Comment:
as commented in another PR, putting this in the middle of a 100-line commit makes it very hard to understand which parts were actually changed...